### DISCUSSION TITLE
please simplified monitor refresh rate what does it do give example please and if i have low cpu and ram since im running via colab whats best settings? ty!

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [crystian/ComfyUI-Crystools#72] [Q&A] [answered]
please simplified monitor refresh rate what does it do give example please and if i have low cpu and ram since im running via colab whats best settings? ty!

**Comment by Amit30swgoh** [comment]
also please show screenshow of settings for old laptop low ram and cpu ty

**Comment by crystian** [comment]
what you talking about? you have this on settings (refresh rate), please check it
it is the same for low ram/cpu or high tech

**Reply by Amit30swgoh** [reply]
yes i know im asking and sorry for my bad english.. im asking which settings are optimal for a bad weak pc please? 
![image](https://github.com/crystian/ComfyUI-Crystools/assets/42430868/e47a3869-0a6d-4606-bc75-e46ac1750338)

**Reply by crystian** [reply, marked as answer]
ah, okok, yes, 0.5 to 1 is ok
